packages/server/controllers/userController.ts

```diff
@@ -21,6 +21,14 @@ export async function getUsersByFirstName(firstName: string) {
   return User.findAll({ where: { firstName } })
 }
 
+export async function getAllUsers() {
```

Comment:
Не верно созданы функции для контролеров. Функции должны принимать два объекта `req` и `res`
```suggestion
export async function getAllUsers(_req: Request, res: Response) {
  const users = User.findAll({
    where: {
      firstName: 'Ivan',
    },
  })

  res.status(200).send(users)
}
```